packages/reassure-logger/src/logger.ts

```diff
@@ -16,9 +16,12 @@ const defaultConfig: LoggerOptions = {
 
 let config: LoggerOptions = { ...defaultConfig };
 
+const colorBrand = chalk.hex(colors.brand);
 const colorError = chalk.hex(colors.error);
 const colorWarn = chalk.hex(colors.warn);
-const colorVerbose = chalk.hex(colors.verbose);
+const colorVerbose = chalk.hex(colors.default);
+
+const prefix: string = colorBrand('reassure: ');
```

Comment:
We agreed with @thymikee that `prefix` is not really useful and pollutes the logs.